properties.md

```diff
@@ -52,44 +55,60 @@ Unlinkability, also known as *blindness*, or *context-hiding* in other works,
+and subsequently in
+[[CKLM13]]({{site.baseurl}}/bibliography.html#cklm13), \[CDHK15\], and
+\[HLR15\].
+
 An anonymous credential scheme where linking the token to a user is protected
-by information-theoretic means has perfect unlinkability, or *strong unlinkabiliy*.
-Studied [by Akagi et al.](https://www.researchgate.net/publication/220797020_An_Efficient_Anonymous_Credential_System).
-When talking about metadata (public or private) it assures that the tokens that
-were issued with the same metadata are indistinguishable to the issuer when
-redeemed.
+by information-theoretic security guarantees has *perfect*, or *strong*
+unlinkability. This property has been formally studied in
+[[AMO08]]({{site.baseurl}}/bibliography.html#amo08), and guarantees that
+attacks on classical cryptosystems by unbounded adversaries will not
+allow linkage of activity in the future.
+
 
 ## Additional Features
 
-Most tokens introduce additional features, that help solving specific problems.
+Anonymous tokens and credentials may introduce additional features, for
+helping thwi solving specific problems.
 
 <details>
 <summary markdown="span">Click for details</summary>
 
 ### Concurrent Security
 
-A system is said to have concurrent security if the signer or issuer of tokens
-or credentials can engage in multiple, concurrent issuances at the same time
-without affecting security. Many schemes have proofs of sequential security,
-wherein at most one issuance flow occurs at a time, yet fail to achieve concurrent
-security. In practice, concurrent security is paramount.
+A system is said to have concurrent security if the signer or issuer of
+tokens or credentials can engage in multiple, concurrent issuances at
+the same time without affecting security. Many schemes only include
+proofs that consider sequential security, wherein at most one issuance
+flow occurs at a time, yet fail to achieve concurrent security. In
+practice, concurrent security is paramount.
 
 ### Single-Show and Multi-Show
 
 Tokens or credentials may be either single- or multi-show, depending on the cryptographic
-construction. For example, tokens derived from the output of an OPRF protocol are
+construction. For example, tokens derived from the output of an [OPRF protocol]({{site.baseurl}}/primitives.html#oprf) are
 typically single-show because spending the same token twice forces the client to
-reveal the same input twice, thereby breaking unlinkability.
+reveal the same input twice, thereby breaking unlinkability. Credentials
+that permit multi-show operations (such as [[CL06]]({{site.baseurl}}/bibliography.html#bcckls08))
 
 ### Revocation
 
 Defined here by [Acar et al.](https://www.iacr.org/archive/pkc2011/65710436/65710436.pdf) and
 implemented using an accumulator scheme with delegatable non-membership proofs.
 Revocation via blocklisting was also implemented by [Tsang et al.](https://www.cs.dartmouth.edu/~sws/pubs/akts07.pdf).
 
-### Public Attributes
+### Associated Data
```

Comment:
How is this different from public attributes?

Reply:
We've made `Public Attributes` a subsection of `Associated data`, mainly to bring in the unlinkability considerations together with this and the private attributes subsection.